PReLU.cu

```diff
@@ -2,12 +2,22 @@
 #include "THCReduce.cuh"
 #include "utils.h"
 
+#include <thrust/functional.h>
+
 // CUDA: grid stride looping
 #define CUDA_KERNEL_LOOP(i, n)                        \
   for (int i = blockIdx.x * blockDim.x + threadIdx.x; \
       i < (n);                                       \
       i += blockDim.x * gridDim.x)
 
+// Use 1024 threads per block, which requires cuda sm_2x or above
+const int CUDA_NUM_THREADS = 1024;
+
+// CUDA: number of blocks for threads.
+inline int GET_BLOCKS(const int N) {
+  return (N + CUDA_NUM_THREADS - 1) / CUDA_NUM_THREADS;
+}
```

Comment:
I was thinking about something fancy. Do you think CUDA_NUM_THREADS should be there too?